strict/src/Data/Strict/Either.hs

```diff
@@ -1,10 +1,7 @@
 {-# LANGUAGE CPP #-}
 {-# LANGUAGE DeriveDataTypeable #-}
 {-# LANGUAGE Safe #-}
-{-# LANGUAGE StandaloneDeriving #-}
-#if __GLASGOW_HASKELL__ >= 706
```

Comment:
I originally added this because the old `strict` package said "Portability :  portable" (i.e. non-GHC). And we still have that in the module docstrings - should we remove it?